docs/spec_draft.md

```diff
@@ -1377,6 +1378,42 @@ implementation-defined.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.log_plus_one
+
+### Semantics
+
+Performs element-wise log plus one operation on `operand` tensor and produces a
+`result` tensor. For floating-point element types, it implements the `logp1`
+operation from the IEEE-754 specification. For complex element types, it
+computes a complex log plus one, with corner cases TBD. Numeric precision is
```

Comment:
@burmako Should we mention it computes the principal value of complex log plus one operation.

Reply:
Good point! Let's do this.

Reply:
Although given that we have other ops with complex inputs which don't call this out, let's keep the text of the spec unchanged and instead open a ticket to consistently call out principal values everywhere.

Reply:
https://github.com/openxla/stablehlo/issues/573